README.md

```diff
@@ -156,6 +156,17 @@ More `<Plug>` mappings can be seen with `:he go-mappings`. Also these are just
 recommendations, you are free to create more advanced mappings or functions
 based on `:he go-commands`.
 
+### Location list navigation
+
+vim-go makes use of Location lists for the output of `:GoLint`, `:GoMetaLinter` etc.
```

Comment:
We use it for all commands, not only for linters. Changing the sentence to describe that aspect would be more useful.